pkg/session/bootstrap_test.go

```diff
@@ -270,12 +270,6 @@ func revertVersionAndVariables(t *testing.T, se sessiontypes.Session, ver int) {
 		// for version <= version195, tidb_enable_dist_task should be disabled before upgrade
 		MustExec(t, se, "update mysql.global_variables set variable_value='off' where variable_name='tidb_enable_dist_task'")
 	}
-	if ver < version212 && ver >= version172 {
```

Comment:
Why Do we delete this logic？Seems this part of logic is used for rollback when the upgrade failed.

Reply:
It is only used in test and the purpose of test is to make sure these DDLs are reentrant.